spec/unit/provider_resolver_spec.rb

```diff
@@ -851,8 +851,7 @@ def stub_service_configs(*configs)
               user: [ Chef::Resource::User::SolarisUser, Chef::Provider::User::Solaris ],
               "5.11" => {
```

Comment:
Is there an omnios or smartos spec in the provider_resolver spec so we can make sure the existing group provider still loads on these platforms?

Reply:
Yeah, there is.

Reply:
👍